tests/test_ops/test_onnx.py

```diff
@@ -58,6 +58,77 @@ def test_nms():
     assert np.allclose(pytorch_score, onnx_score, atol=1e-3)
 
 
+def test_softnms():
+    if not torch.cuda.is_available():
+        return
+    from packaging import version
+    from mmcv.ops import soft_nms, get_onnxruntime_op_path
+
+    # only support pytorch >= 1.7.0
+    if version.parse(torch.__version__) < version.parse('1.7.0'):
+        return
+
+    # only support onnxruntime >= 1.5.0
```

Comment:
Actually, there is no onnxruntime of 1.5.0, check [here](https://github.com/microsoft/onnxruntime/tags). Please update the version.